modules/evm/src/runner/stack.rs

```diff
@@ -551,7 +551,14 @@ impl<'vicinity, 'config, T: Config> BackendT for SubstrateStackState<'vicinity,
 	}
 
 	fn code(&self, address: H160) -> Vec<u8> {
-		Pallet::<T>::code_at_address(&address).into_inner()
+		let code = Pallet::<T>::code_at_address(&address).into_inner();
+		log::debug!(
```

Comment:
why log on getter?

Reply:
If the contract does not exist, evm will return revert without error msg. 
It's hard to debug, so log here.